lib/active_merchant/billing/gateways/cyber_source.rb

```diff
@@ -1053,6 +1053,8 @@ def commit(request, action, amount, options)
         message = message_from(response)
         authorization = success || in_fraud_review?(response) ? authorization_from(response, action, amount, options) : nil
 
+        auto_void?(authorization_from(response, action, amount, options), response, options)
```

Comment:
Should we update the message to reflect that we voided the transaction because of the error code?